test/explorer/series/struct_test.exs

```diff
@@ -4,6 +4,23 @@ defmodule Explorer.Series.StructTest do
   alias Explorer.Series
 
   describe "from_list/2" do
+    test "allows struct of all nil value" do
```

Comment:
What about a list of lists of nils?

    Series.from_list([[nil, nil], [nil, nil, nil]])